util/log/src/main/scala/sbt/ConsoleLogger.scala

```diff
@@ -35,6 +35,19 @@ object ConsoleLogger {
   def isEscapeTerminator(c: Char): Boolean =
     c >= '@' && c <= '~'
 
+  /**
+   * Test if the character AFTER an ESC is the ANSI CSI, which means more than two characters until terminatioin.
+   */
+  def isCSI(c: Char): Boolean = c == '['
+
+  /**
+   * Tests whether or not a character needs to immediately terminate the ANSI sequence.
+   */
+  def isSimpleEscapeTerminator(c: Char): Boolean = {
+    val n = c.toInt
+    (n >= 64) && (n <= 95)
```

Comment:
http://en.wikipedia.org/wiki/ANSI_escape_code#Sequence_elements

> Escape sequences start with the character ESC (ASCII decimal 27/hex 0x1B/octal 033). For two character sequences, the second character is in the range ASCII 64–95 (@ to _).

ok. So this is non-CSI case. Having `isEscapeTerminator` and `isSimpleEscapeTerminator` without any explanation is a bit confusing. Also these are all public functions?